risc0/zkvm/src/guest/env.rs

```diff
@@ -199,6 +213,11 @@ impl std::error::Error for VerifyError {}
 /// Verify that there exists a valid receipt with the specified
 /// [ReceiptClaim].
 ///
+/// Calling this function in the guest is logically equivalent to verying a receipt with the same
```

Comment:
```suggestion
/// Calling this function in the guest is logically equivalent to verifying a receipt with the same
```

Reply:
Resolved in https://github.com/risc0/risc0/pull/1274/commits/5365a056a93c8f585e9d6c8aea739d3d011a3291